h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -504,7 +516,28 @@ private Model buildModel(final MP params, Grid<MP> grid, int paramsIdx, String p
     }
 
     // Is there a model with the same params in the DKV?
-    @SuppressWarnings("unchecked") final Key<Model>[] modelKeys = KeySnapshot.globalSnapshot().filter(new KeySnapshot.KVFilter() {
+    Key<Model>[] modelKeys = findModelsByChecksum(checksum);
```

Comment:
this whole block is just refactoring, correct?

Reply:
yes, no other changes